imports/plugins/core/orders/server/no-meteor/schemas/schema.graphql

```diff
@@ -395,6 +410,27 @@ type OrderNote {
   updatedAt: DateTime!
 }
 
+"A refund of a payment on an order"
+type Refund implements Node {
+  "The refund ID"
+  _id: ID!
+
+  "The amount of the refund"
+  amount: Money!
+
+  "The date and time at which the refund was created"
+  createdAt: DateTime!
+
+  "The display name of the payment refunded to"
+  paymentDisplayName: String!
+
+  "The reason for the refund"
+  reason: String
+
+  "The type of refund"
+  type: RefundType
```

Comment:
Do we use/show this anywhere? It's unclear to me what it's for so maybe we should omit it from the GraphQL for now?

Reply:
No, we don't use it, will remove.